index.html

```diff
@@ -127,77 +127,142 @@ <h2>Dictionary {{RTCRtpEncodingParameters}} Members</h2>
-          simulcast encodings on a single RTP stream. The
-          <a>user agent</a>  MUST return {{OperationError}} in
-          {{RTCRtpSender/setParameters()}} or
-          {{RTCPeerConnection/addTransceiver()}} in response to
-          an attempt to simultaneously utilize both simulcast transport
-          techniques.
+          {{RTCPeerConnection/addTransceiver()}}, an "S mode" cannot
+          be requested. The browser may only be configured to send
+          simulcast encodings with multiple SSRCs and RIDs, or
+          alternatively, to send all simulcast encodings on a single
+          RTP stream. Simultaneously using both simulcast transport
+          techniques is not permitted. 
         </p>
+      </section>
+      <section id="setparameters">
+       <h3>{{RTCRtpSender/setParameters()}}</h3>
+         <p>
+           [[!WEBRTC]] Section 5.2 describes validation of <var>parameters</var>
+           within {{RTCRtpSender/setParameters()}}. Add the following to the 
+           conditions (step 4 within step 6) for an {{InvalidModificationError}}:
+           <ol>
+             <li>
+               <var>encodings</var> contains an encoding whose
+               {{RTCRtpEncodingParameters/scalabilityMode}} value is
+               not supported by any codec in <var>codecs</var>.
+             </li>
+             <li>
+               <var>N</var> is greater than 1, and <var>encodings</var>
+               contains an encoding whose
+               {{RTCRtpEncodingParameters/scalabilityMode}} value
+               represents an "S mode".
+             </li>
+           </ol>
+         </p>
+      </section>
+      <section id="getparameters">
+       <h3>{{RTCRtpSender/getParameters()}}</h3>
+         <p>
+           To determine whether {{RTCRtpEncodingParameters/scalabilityMode}}
+           values requested via {{RTCPeerConnection/addTransceiver()}} or
+           {{RTCRtpSender/setParameters()}} have been applied,
+           an application can call the {{RTCRtpSender.getParameters()}} method
+           after negotiation has completed and the sending codec has been
+           determined. If the configuration is not satisfactory,
+           {{RTCRtpSender/setParameters()}} can be used to change it.
+         </p>
+         <p>
+           {{RTCRtpSender/getParameters()}} returns the currently
+           configured {{RTCRtpEncodingParameters/scalabilityMode}} value for each
+           <var>encoding</var>.
+         </p>
+         <p>
+           Before negotiation completes, {{RTCRtpSender/getParameters()}} returns the
+           {{RTCRtpEncodingParameters/scalabilityMode}}
+           values requested in {{RTCRtpSender/setParameters()}} or
+           {{RTCPeerConnection/addTransceiver()}}, assuming that those methods completed
+           successfully.
+         </p>
+         <p>
+           If no {{RTCRtpEncodingParameters/scalabilityMode}} value was provided for
```

Comment:
IMHO Before negotiation, I think it is better to not provide any value if the `scalabilityMode` is not set. Maybe we should specify on the codec capabilities what is the default `scalabilityMode`   (if any) for each codec instead.

After negotiation the `scalabilityMode`  will reflect the one currently in use.

We also need to specify if "L1T1" is the mode for all codecs not supporting temporal or spatial scalability.

Reply:
I'm coming around to your point of view.

Reply:
I think "no value until we've started to use one or it's explicitly set" makes more sense.

Reply:
Fixed.